package/Dockerfile

```
@@ -15,10 +15,13 @@ RUN mkdir -p /go/src/github.com/k3s-io/helm-set-status && \
     make -C /go/src/github.com/k3s-io/helm-set-status install
 
 FROM alpine:3.12
-RUN apk add -U --no-cache ca-certificates jq bash git
-COPY --from=extract /usr/bin/helm_v2 /usr/bin/helm_v3 /usr/bin/tiller /usr/bin/
-COPY --from=plugins /root/.local/share/helm/plugins/ /root/.local/share/helm/plugins/
+RUN apk add -U --no-cache ca-certificates jq bash git && \
+    adduser -D -u 1000 -s /bin/bash klipper-helm
+WORKDIR /home/klipper-helm
+COPY --chown=1000:1000 --from=extract /usr/bin/helm_v2 /usr/bin/helm_v3 /usr/bin/tiller /usr/bin/
```

This is going into /usr/bin and should still be root owned. Only the files going into the home directory need to be owned by the user.
```suggestion
COPY --from=extract /usr/bin/helm_v2 /usr/bin/helm_v3 /usr/bin/tiller /usr/bin/
```

---

GAH! That's embarrassing. Sorry about that. Fixed.